analytics_platform_concourse_webhook_dispatcher/server.py

```diff
@@ -5,14 +5,17 @@
 from sanic import Sanic
 from sanic.exceptions import abort
 from sanic.response import json
+from fly import Fly
 
 from analytics_platform_concourse_webhook_dispatcher.config import Config
-from analytics_platform_concourse_webhook_dispatcher import events, concourse
+from analytics_platform_concourse_webhook_dispatcher import events
 from .signature import make_digest
 
 app = Sanic()
 app.config.from_object(Config)
 
+fly_lock = asyncio.Lock()
+
 
 @app.listener("before_server_start")
 async def aiohttp_setup(app, loop):
```

Comment:
could delete this whole method if you're not doing any http calls

Reply:
and remove aiohttp dependency

Reply:
as in remove
```python
@app.listener("before_server_start")
async def aiohttp_setup(app, loop):
    # setup the session
    timeout = aiohttp.ClientTimeout(total=10)
    session = aiohttp.ClientSession(loop=loop, timeout=timeout)
    app.http = session


@app.listener("after_server_stop")
async def aiohttp_teardown(app, loop):
    await app.http.close()
```

Reply:
yep